### PR TITLE
fix: compile errors

### PR DESCRIPTION
**Pull request by larstobi**
fix: compile errors Parameter 'cmd' implicitly has an 'any' type.
fix: arg of type unknown is not assignable to parameter of type x
fix: catch clause variable cannot have type annotation

```
@zwave-js/config: src/Devices.ts(456,7): error TS7006: Parameter 'cmd' implicitly has an 'any' type.
@zwave-js/config: src/Devices.ts(464,10): error TS7006: Parameter 'arg' implicitly has an 'any' type.
@zwave-js/config: src/Devices.ts(652,6): error TS7006: Parameter 'opt' implicitly has an 'any' type.
@zwave-js/config: src/Meters.ts(39,36): error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'Record<string, any>'.
@zwave-js/config: src/DeviceClasses.ts(215,30): error TS2339: Property 'every' does not exist on type 'never'.
@zwave-js/config: src/DeviceClasses.ts(246,31): error TS2339: Property 'every' does not exist on type 'never'.
@zwave-js/config: src/DeviceClasses.ts(328,30): error TS2339: Property 'every' does not exist on type 'never'.
@zwave-js/config: src/DeviceClasses.ts(363,31): error TS2339: Property 'every' does not exist on type 'never'.
```

### CODE REVIEW

File: packages/config/src/Devices.ts [hunk 1]
```diff
@@ -649,7 +649,7 @@ Parameter #${parameterNumber}: allowManualEntry must be a boolean!`,
 		if (
 			isArray(definition.options) &&
 			!definition.options.every(
-				(opt) =>
+				(opt: any) =>
 					isObject(opt) &&
 					typeof (opt as any).label === "string" &&
 					typeof (opt as any).value === "number",
```
Comment (AlCalzone) [inline packages/config/src/Devices.ts]: The `as any` here should now be unnecessary.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, just a little detail

File: packages/config/src/Devices.ts (hunk 1 above)
Comment (larstobi) [reply packages/config/src/Devices.ts]: I can remove it

Comment (larstobi) [reply packages/config/src/Devices.ts]: I wasn't able to remove it. it says:
```
Devices.ts:652:6 - error TS7006: Parameter 'opt' implicitly has an 'any' type.
652  			(opt) =>
     			 ~~~
```

**Comment by larstobi** [comment]
I converted it to a draft. I need to fix linting issues as well...

**Comment by larstobi** [comment]
This now builds on my machine. I'm not able to reproduce the errors form the lint stage.

**Comment by AlCalzone** [comment]
I've had some problems too in https://github.com/zwave-js/node-zwave-js/pull/1192
Got around them with
```
npm ci
```
to make sure I had the same versions locally.

**Comment by AlCalzone** [comment]
@larstobi I ran across the errors in another PR and fixed them slightly differently. I've merged those changes and fixed some nits - if this builds, LGTM.

**Review by AlCalzone — APPROVED** [review]

File: packages/config/src/DeviceClasses.ts [hunk 2]
```diff
@@ -211,11 +211,10 @@ export class GenericDeviceClass {
 
 		if (definition.supportedCCs != undefined) {
 			if (
-				!isArray(definition.controlledCCs) ||
-				(isArray(definition.supportedCCs) &&
-					!definition.supportedCCs.every(
-						(cc: any) => typeof cc === "string",
-					))
+				!isArray(definition.supportedCCs) ||
```
Comment (larstobi) [inline packages/config/src/DeviceClasses.ts:214]: Well, d'oh, I should have seen this one. 😂

**Comment by larstobi** [comment]
@AlCalzone Looks like it builds nicely :-)

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**